RELEASE-NOTES.txt

```diff
@@ -10,6 +10,7 @@
 * [*] Embed block: Fix URL update when edited after setting a bad URL of a provider. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4002]
 * [**] Users can now contact support from inside the block editor screen. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3975]
 * [**] Site Comments: when editing a Comment, the author's name, email address, and web address can now be changed. (https://github.com/wordpress-mobile/WordPress-Android/pull/15402)
+* [**] Block editor: Support articles available within the block editor help menu [#15421]
```

Comment:
👋 @dcalhoun! My understanding is that by removing the `canViewEditorOnboarding()` conditional it's the whole of "Help & Support" menu that becomes available, not just the support articles button, right? If so, what do you think about making the release notes about it more about the whole functionality? Something like: `Block editor: Help menu with guides about how to work with blocks[#15421]` perhas. WDYT?

Reply:
Makes sense. I have addressed this in https://github.com/wordpress-mobile/WordPress-Android/pull/15421/commits/fae3f8a49ade1030c16b299c940b77f9d5dccb91.